app/views/coronavirus/pages/show/_timeline_entries.html.erb

```diff
@@ -4,7 +4,7 @@
      id: timeline_entry.id,
      field: timeline_entry.heading,
      value: timeline_entry.national_applicability_text,
-     edit: { href: edit_coronavirus_page_timeline_entry_path(@page.slug, timeline_entry) },
+     edit: { href: edit_coronavirus_page_timeline_entry_path(@page.slug, timeline_entry), link_text: "Edit" },
```

Comment:
Could this text be moved to the locale files to keep these in a consistent location, as is the case for the 'Reorder' label below (`t("coronavirus.pages.show.timeline_entries.reorder")`)?

Reply:
I think the other summary lists on the page need to be updated too. i.e. The guidance and support list, and the announcements list: https://collections-publisher.integration.publishing.service.gov.uk/coronavirus/landing